packages/core-components/src/components/Select/Select.tsx

```diff
@@ -218,15 +218,21 @@ export function SelectComponent(props: SelectProps) {
           renderValue={s =>
             multiple && (value as any[]).length !== 0 ? (
               <Box className={classes.chips}>
-                {(s as string[]).map(selectedValue => (
-                  <Chip
-                    key={items.find(el => el.value === selectedValue)?.value}
-                    label={items.find(el => el.value === selectedValue)?.label}
-                    clickable
-                    onDelete={handleDelete(selectedValue)}
-                    className={classes.chip}
-                  />
-                ))}
+                {(s as string[]).map(selectedValue =>
+                  items.find(el => el.value === selectedValue) ? (
+                    <Chip
+                      key={items.find(el => el.value === selectedValue)?.value}
+                      label={
+                        items.find(el => el.value === selectedValue)?.label
+                      }
+                      clickable
+                      onDelete={handleDelete(selectedValue)}
+                      className={classes.chip}
+                    />
+                  ) : (
+                    false
+                  ),
+                )}
```

Comment:
Makes senes! Not caused by you, but as this code is already touched, would you be up for simplifying this by running `.find` only once instead of 3 times? :)